.eslintrc.json

```diff
@@ -6,6 +6,7 @@
   "rules": {
     "@typescript-eslint/no-namespace": "off",
     "@typescript-eslint/strict-boolean-expressions": "off",
-    "@typescript-eslint/no-misused-promises": "off"
+    "@typescript-eslint/no-misused-promises": "off",
+    "@typescript-eslint/no-invalid-void-type": "off"
```

Comment:
Is it really necessary to disable this linter rule?

Reply:
Yes, to type Result param as void without hardcode like:  

```js
namespace UseCase {
  type Result = void
}
```

To use `Promise<UseCase.Result>`, not `Promise<void>`.